### ISSUE TITLE
MacOS (Windows/Linux also) Build System

### ISSUE THREAD

**Issue by Zayelion** [SalvationDevelopment/YGOPro-Support-System#145]
The repo runs out the box for linux and windows, just install github, and clone the repo then wait for the server to synch after starting the client.

Mac requires some wrapping, create a build script for creating that after all the files are in place.

**Comment by IngwiePhoenix** [comment]
What exactly does need to be „built“?

**Comment by Zayelion** [comment]
Pull the launcher down from github, the turn it into launcher.app

**Comment by IngwiePhoenix** [comment]
Gotcha. Gonna do this tomorrow. Currently working on learning math for an exam tomorrow. =D
Should be really easy.

**Comment by Zayelion** [comment]
btw keep an eye on https://github.com/SalvationDevelopment/YGOPro-Support-System/issues/146

**Comment by Zayelion** [comment]
Any update to this.... angry users are attacking.

**Comment by IngwiePhoenix** [comment]
Well, bad news is, that this is not something to be fixed.

The native client is simply broken on OS X. I tried to look into it, but i can not determine the actual cause of the problem. In fact I have invited a friend to look at it and he did not see anything. So I defined this is as unfixable. The code isnt very well documented to give any clues either.

I thought it was working fine on Windows?

I did not test Linux myself. But I am installing Ubunto today’s afternoon.

Another issue with the actual native client is how it is being built. its a struggle with all the pre-built libs and blah. Its one reason why I was driven away from ever using IrrLicht or irrKlang. x.x

But yeah, Mac OS X does not seem like anything I can fix you, sorry.
